### ISSUE TITLE
Windows guide broken image in 3.2

### ISSUE THREAD

**Issue by ncbaratta** [red-hat-openjdk/docs#31]
QE by @sparkoo 

https://red-hat-openjdk.github.io/docs/windows/index.html#setup-_JDKMissionControl_procedure_windows

**Comment by ncbaratta** [comment]
@robin-owen do you have any ideas on this one.  When you see the doc here: https://github.com/red-hat-openjdk/docs/blob/master/modules/proc-windows-setup-jdkmissioncontrol.adoc the image shows, but when you view the built guide it does not.  Could it be the build script?

**Comment by kowen-rh** [comment]
Yes, this would be related to the build script not appropriately copying the `images/` directory to `docs/images`, I believe. This is difficult for me to test locally, but I'm pretty sure we just need to add `cp -r images/ docs/images/` to the `build.sh` script in order for GitHub Pages to pick it up.

**Comment by ncbaratta** [comment]
If you find some time can you give that a whirl? Or we can just leave it be and when we build on the portal we can troubleshoot then.

**Comment by kowen-rh** [comment]
> If you find some time can you give that a whirl?

Yes, absolutely! Apologies for the late response. Taking some time to look into it now. :)
